templates/docs/examples/patterns/grid/_aside.html

```diff
@@ -0,0 +1,64 @@
+<form>
```

Comment:
When moving examples from `patterns` to `layouts` this should be moved as well.

Reply:
done